modules/admin-credentials-root-secret-formats.adoc

```diff
@@ -102,16 +89,61 @@ metadata:
   namespace: kube-system
   name: gcp-credentials
 stringData:
-  service_account.json: <ServiceAccount>
+  service_account.json: <base64-encoded_service_account>
 ----
-endif::google-cloud-platform[]
 
-ifeval::["{context}" == "manually-creating-iam-aws"]
-:!aws:
-endif::[]
-ifeval::["{context}" == "manually-creating-iam-azure"]
-:!azure:
-endif::[]
-ifeval::["{context}" == "manually-creating-iam-gcp"]
-:!google-cloud-platform:
+ifdef::passthrough[]
+
+.{rh-openstack-first} secret format
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: kube-system
+  name: openstack-credentials
+data:
+  clouds.yaml: <base64-encoded_cloud_creds>
+  clouds.conf: <base64-encoded_cloud_creds_init>
+----
+
+.{rh-virtualization-first} secret format
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: kube-system
+  name: ovirt-credentials
+data:
+  ovirt_url: <base64-encoded_url>
+  ovirt_username: <base64-encoded_username>
+  ovirt_password: <base64-encoded_password>
+  ovirt_insecure: <base64-encoded_insecure>
+  ovirt_ca_bundle: <base64-encoded_ca_bundle>
+----
+
+.VMware vSphere secret format
+
+[source,yaml]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  namespace: kube-system
+  name: vsphere-creds
+data:
+ {{VCenter.username}}: <base64-encoded_username>
```

Comment:
@jeana-redhat, so this, I wonder about, because I forget how variable interpolation works with k8s manifests, so I don't know if this is actually operative or not here. Like maybe this needs to be part of a template to work.

Reply:
thanks! I have asked Akhil about this :+1:

Reply:
Fantastic! Thank you @jboxman-rh for your eagle-eyed review, and @akhil-rane for digging in to verify it :handshake: Updated accordingly.